EIPS/eip-6969.md

```diff
@@ -0,0 +1,76 @@
+---
+eip: 6969
+title: Submit Block RPC
+author: Zachary Belford (@belfordz)
+status: Draft
+type: Standards Track
+category: Interface
+created: 2019-06-26
+---
+
+## Simple Summary
+Add an RPC method to submit a new block, instead of having to use submitWork.
+
+## Abstract
+At the moment, there is no high level facility through which an external-to-geth system can submit a newly mined block. Mining pools rely on submitWork, which implies that the provider of the submitted work is not able to, at the time of submission, provide an address to receive the payout. 
+
+By adding a JSON-RPC method that accepts a fully formed new block to submit instead of having to use `submitWork`, we will drastically simplify the process and software required to build a highly scalable mining pool. 
+
+## Motivation
+I would like to be able to submitWork, but have the address used be one other than the etherbase of the node the work is being submitted by.
+
+## Specification
+Below is the method specified using [OpenRPC](https://spec.open-rpc.org) spec.
```

Comment:
Can you add a traditional description (too) which is used by the other EIPs for consistency?

Reply:
I've added more of a description. I'm very open to suggestion / collaboration on this EIP. Let me know if this works. Im happy to try again :).